chainer/datasets/svhn.py

```diff
@@ -12,7 +12,7 @@
 from chainer.datasets import tuple_dataset
 
 
-def get_svhn(withlabel=True, scale=1., dtype=numpy.float32,
+def get_svhn(withlabel=True, scale=1., add_extra=False, dtype=numpy.float32,
```

Comment:
Please do not insert arguments in-between, as this breaks backward compatibility.
Append it after `label_dtype`.